cmd/dist/labels.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	digest "github.com/opencontainers/go-digest"
+	"github.com/urfave/cli"
+)
+
+var labelCommand = cli.Command{
+	Name:        "label",
```

Comment:
I would like to go to this, so it is okay to say no and leave it! `label` is a verb.